tests/source/github_org_test.py

```diff
@@ -21,5 +21,7 @@ def repos_response(mock_urlopen):
 def test_list_repos(repos_response):
     settings = Settings('key', 'sass')
     ret = list_repos(settings)
-    expected = {'sass/libsass-python': 'git@github.com:sass/libsass-python'}
```

Comment:
this is not ideal, can we strip `.git`?

Reply:
sure. could you explain what makes it not ideal?

Reply:
there are other scripts which depend on that value being as it is, both in this repository and in others

changing the value will also create an inconsistent state for those upgrading from an older version

and lastly, `.git` is unnecessary to clone